topics/imaging/tutorials/2D-spot-detection/tutorial.md

```diff
@@ -88,15 +90,16 @@ and selected the 10th z-stack for analysis.
 
 # Improve image contrast 
 
-Your image should look something like this:
+When inspecting the image visually, we get the impression that the image is entirely black:
 
 ![2025_pre_clahe.png](../../images/2d_spot_detection/2025_pre_clahe.png){: width="75%"}
 
 It's not the best quality for a first visual inspection!
 
 
-As described in [a previous Galaxy Tutorial]{% link topics/imaging/tutorials/imaging-introduction/tutorial.md %}, the original image is 16-bit and the intensity values are spread
-over a larger range than the display can render. 
+As described in the Galaxy Imaging Introduction Tutorial, the original image is 16-bit and the intensity values are spread
```

Comment:
Did you remove the link on intention? If so, nevermind. I just think it was quite useful :)